packages/cms-editor/src/translation/cms-editor/de.json

```diff
@@ -20,11 +20,20 @@
       "createTooltip": "Halte {{modifier}} um ein weiteres Inhaltsobjekt zu erstellen.",
       "description": "Wählen Sie den Namen, den Namensbereich und die Werte des Inhaltsobjekts, das Sie hinzufügen möchten.",
       "title": "Inhaltsobjekt hinzufügen"
+    },
+    "languageTool": {
+      "description": "Bearbeite die Sprachen des CMS.",
+      "label": {
+        "defaultLanguage": "Standardsprache"
+      },
+      "save": "Speichern",
```

Comment:
I think save shouldbe or is in the common.label.
Or to be clear, even if we don't have this common namespace any longer, I think we should still hold the same labels under the same key. Because I'm not sure if WebLate also shows similar translations if the labels are not under the same key. So for those cases I think we should still save common stuff under a common prefix, so `common.label.save`